src/reduceRight.js

```diff
@@ -37,7 +37,7 @@ var _curry3 = require('./internal/_curry3');
 module.exports = _curry3(function reduceRight(fn, acc, list) {
   var idx = list.length - 1;
   while (idx >= 0) {
-    acc = fn(acc, list[idx]);
+    acc = fn(list[idx], acc);
     idx -= 1;
   }
   return acc;
```

Comment:
OK, I'll update the comment. The procedure of `reduceRight` should be as follows:
```js
R.reduce((a, b) => a - b, 0, [1, 2, 3, 4]); //=> (((0 - 1) - 2) - 3) - 4) => -10
R.reduceRight((a, b) => a - b, 0, [1, 2, 3, 4]); //=>  (1 - (2 - (3 - (4 - 0)))) => -2
```